services/facade/server/models/account.js

```diff
@@ -14,36 +47,48 @@ module.exports = function(Account) {
     transactionService.getFunction = getFunction;
   });
 
-  function getFunction(model, method) {
-    let functionName = model + '_' + method;
-    return this.createModel(model, {})[functionName];
+  function findTransaction(input) {
+    let find = transactionService.getFunction('Transaction', 'findById');
+    return new Promise(function (resolve, reject) {
+      find(input, function(err, data) {
```

Comment:
I think this is sort of an anti-pattern in Bluebird, use `promisify` instead - see http://bluebirdjs.com/docs/api/promise.promisify.html